src/GeolocationContext.tsx

```diff
@@ -18,6 +25,8 @@ import {updateMetadata as updateChatUserMetadata} from './chat/metadata';
 import {useAppStateStatus} from './utils/use-app-state-status';
 import {GeoLocation} from '@atb/favorites';
 import {dictionary, useTranslation} from '@atb/translations';
+import {useOnResume} from './utils/use-on-resume';
+import {useOnPause} from './utils/use-on-pause';
```

Comment:
Other places we have used the term `active`, like in the hook `useIsFocusedAndActive`. Is it possible to align the terms used in our app?

Reply:
Solved by using appStatus instead of useOnResume and useOnPause hooks